cdap-watchdog/src/main/java/co/cask/cdap/logging/framework/CDAPLogAppender.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.logging.framework;
```

Comment:
there are existing `LogAppender` class in `co.cask.cdap.logging.appender`, i thought we can keep this in framework and when we are ready to remove old classes, we can refactor the package name. I can add a todo for now. is that okay ?